src/parameterFactory.js

```diff
@@ -0,0 +1,124 @@
+import {
+  DateInput,
+  EmailInput,
+  NullableBooleanInput,
+  NumberInput,
+  TextInput,
+} from 'react-admin';
+import React from 'react';
+
+function guessType(parameter, fields) {
```

Comment:
I'm not a fond of this function because it's hardcoded for specific API Platform values, couldn't we use the Hydra metadata instead? The type is provided.